manifests/2.7.0/opensearch-2.7.0.yml

```diff
@@ -175,7 +175,7 @@ components:
       - gradle:properties:version
   - name: performance-analyzer
     repository: https://github.com/opensearch-project/performance-analyzer.git
-    ref: 2.x
+    ref: '2.7'
```

Comment:
PA is missing this branch, Hence CI is failing. Please create 2.7 branch

Reply:
Yes thanks @gaiksaya. I have already reached out to @khushbr offline and she's working on this.

Reply:
@gaiksaya @zelinh This has been fixed now. Can we retry the Github checks?